### PR TITLE
replace react-native-video with expo-av

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/unfurl/image/video.native.tsx [hunk 1]
```diff
@@ -11,13 +11,20 @@ type State = {
...
       return
     }
-    this.setState({playingVideo: !this.state.playingVideo})
+    this.setState(({playingVideo}) => {
+      if (!this.videoRef.current) {
+        return
+      }
+      playingVideo ? this.videoRef.current.pauseAsync() : this.videoRef.current.playAsync()
+      return {playingVideo: !playingVideo}
+    })
```
[the remark this answers is not in the document]
Comment (mmaxim) [reply shared/chat/conversation/messages/wrapper/unfurl/image/video.native.tsx]: Prefer to not refactor this whole component for this PR

**Comment by mmaxim** [comment]
Did you try unfurls of various types that have video?

File: shared/chat/conversation/messages/attachment/image/image-render.native.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as Kb from '../../../../../common-adapters/mobile.native'
 import * as Styles from '../../../../../styles'
 import logger from '../../../../../logger'
 import {Props} from './image-render.types'
-import Video from 'react-native-video'
+import {Video as ExpoVideo} from 'expo-av'
```
[the remark this answers is not in the document]
Comment (songgao) [reply shared/chat/conversation/messages/attachment/image/image-render.native.tsx]: Oh, I guess this one doesn't need rename. Will fix.

**Comment by songgao** [comment]
> Did you try unfurls of various types that have video?

@mmaxim I tried video attachment and giphy unfurl. Anything else I should test?

**Comment by mmaxim** [comment]
Should be good

**Pull request merged**